src/viur/shop/skeletons/article.py

```diff
@@ -88,17 +88,14 @@ def shop_is_low_price(self) -> BooleanBone:
     def shop_price_(self) -> Price:
         return Price.get_or_create(self)
 
-    shop_price = RawBone(  # FIXME: JsonBone doesn't work (https://github.com/viur-framework/viur-core/issues/1092)
+    shop_price = JsonBone(
         compute=Compute(lambda skel: skel.shop_price_.to_dict(), ComputeInterval(ComputeMethod.Always))
     )
-    shop_price.type = JsonBone.type
-
-    shop_shipping = RawBone(  # FIXME: JsonBone doesn't work (https://github.com/viur-framework/viur-core/issues/1092)
+    shop_shipping = JsonBone(
         compute=Compute(
             lambda skel: make_json_dumpable(SHOP_INSTANCE.get().shipping.choose_shipping_skel_for_article(skel)),
             ComputeInterval(ComputeMethod.Always)),
     )
-    shop_shipping.type = JsonBone.type
     """Calculated, cheapest shipping for this article"""
```

Comment:
This was only fixed in `viur-core==3.7.0-devX`. As long as there is no stable release of 3.7.0, we should leave it as it is. But even then we'll have to see how many core versions we want to support with viur-shop. In general, I would like 3.6 to be supported so that I am not immediately forced to update my customer project.